hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ContainerKeyInfoResponse.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.hadoop.ozone.debug;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Class for response for container key scanner.
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ContainerKeyInfoResponse {
+
+  private final long keysProcessed;
+  private final Map<Long, List<ContainerKeyInfo>> containerKeys;
+
+  public ContainerKeyInfoResponse(
+      long keysProcessed, Map<Long, List<ContainerKeyInfo>> containerKeys) {
+    this.keysProcessed = keysProcessed;
+    this.containerKeys = containerKeys;
+  }
+
+  public long getKeysProcessed() {
+    return keysProcessed;
+  }
+
+  public Map<Long, List<ContainerKeyInfo>> getContainerKeys() {
+    return containerKeys;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
```

Comment:
@hemantk-12 the class must also be `final` to be OK to use `instanceof` in `equals()`.  Otherwise equality with an instance of a subclass may not be symmetric (i.e. result may vary according to which instance is `equals(other)` invoked on.

Reply:
Fixed.